src/__tests__/components/__snapshots__/MenuTab.test.js.snap

```diff
@@ -100,6 +98,7 @@ exports[`MenuTab should render with loading props 1`] = `
       "icon": "#FFFFFF",
       "iconDeactivated": "rgba(255, 255, 255, .75)",
       "iconLoadingOverlay": "rgba(255, 255, 255, .75)",
+      "iconServerBaseUri": "https://content.edge.app",
```

Comment:
These changes belong in the parent feature branch (future commit). I updated the [patch/liquidity-pool-ref](https://github.com/EdgeApp/edge-react-gui/pull/3310).